hooks/build

```diff
@@ -10,6 +10,6 @@ docker pull $BASE_IMAGE
 BUILD_DATE="$(date --rfc-3339 seconds)"
 VERSION=$(grep VERSION ../setup.py | cut -f3 -d" " | tr -d '"')
 printenv
-docker build --build-arg BUILD_DATE="${BUILD_DATE}" --build-arg SOURCE_COMMIT="${GIT_SHA1}" --build-arg VERSION=$VERSION -t $IMAGE_NAME ..
+docker build --build-arg BUILD_DATE="${BUILD_DATE}" --build-arg SOURCE_COMMIT="${GIT_SHA1}" --build-arg VERSION=$VERSION --tag $IMAGE_NAME --file Dockerfile ..
```

Comment:
Setting defaults twice is always a bad idea.

Reply:
which one do you mean?

Reply:
The default, you've actually set: `--file Dockerfile` This is the default and unneccessary.

Reply:
no, i thougth that too, but in my environment it's necessary to use trhat option explicitly when `..` is used as context root.

Reply:
Oh, sh***. I haven't said anything 😉